include/pisa/command/invert.hpp

```diff
@@ -0,0 +1,40 @@
+#pragma once
+
+#include <thread>
+#include <variant>
+
+#include <CLI/CLI.hpp>
```

Comment:
We should not have a CLI dependency here in my opinion...

Reply:
We could get away with this by moving Settings (see my other comment for that) to the other header, and `command/invert.hpp` or in that case even just `cli.hpp` file or similar, has the parsing function that is being tested. So no CLI dependency but we include the original header for testing parsing.